packages/api/endpoints/dead-letter-archive.js

```diff
@@ -11,27 +11,39 @@ async function postRecoverCumulusMessages(req, res) {
   const stackName = process.env.stackName;
   const systemBucket = process.env.system_bucket;
 
-  const { bucket, path } = (req.body === undefined ? {} : req.body);
+  const {
+    bucket,
+    path,
+    batchSize = 1000,
+    concurrency = 30,
+    maxDbPool = 50,
+  } = req.body ?? {}; // TODO ZOD!  KNEEL BEFORE ZOD
```

Comment:
😅  Reviewers: apologies, I just realized I did not do this/missed the comment before putting up for review.       I think review is still valuable, but be aware there will be a follow on to add zod/arg parsing.

Reply:
c28cf580b75